tests/playwright/src/model/preferences-extension-ai-lab-page.ts

```diff
@@ -0,0 +1,50 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Locator, Page } from '@playwright/test';
+import { expect as playExpect, PreferencesPage } from '@podman-desktop/tests-playwright';
+
+export class ExtensionAILabPreferencesPage extends PreferencesPage {
+  public static readonly tabName = 'Extension: AI Lab';
+  readonly heading: Locator;
+
+  constructor(page: Page) {
+    super(page);
+    this.heading = page.getByRole('heading', { name: ExtensionAILabPreferencesPage.tabName });
+  }
+
+  async waitForLoad(): Promise<void> {
+    await playExpect(this.heading).toBeVisible();
+  }
+
+  public async disableGPUPreference(): Promise<void> {
+    await this.content
+      .getByRole('checkbox', { name: 'Experimental GPU support for inference servers' })
+      .uncheck({ force: true });
+  }
+
+  public async enableGPUPreference(): Promise<void> {
+    await this.content
+      .getByRole('checkbox', { name: 'Experimental GPU support for inference servers' })
+      .check({ force: true });
```

Comment:
You should assert that the checkbox is checked after setting it. Same for disabling.